tests/ecmp/test_fgnhg.py

```diff
@@ -1,9 +1,13 @@
 import pytest
-
+import difflib
 import time
 import logging
 import ipaddress
 import json
+import collections
+from tabulate import tabulate
+from tests.common.reboot import reboot
+from multiprocessing.pool import ThreadPool, TimeoutError
```

Comment:
@kktheballer why do we need `TimeoutError`?

Reply:
removed

Reply:
taken out